src/utils/ExtrusionJunction.h

```diff
@@ -47,6 +47,14 @@ struct ExtrusionJunction
     ExtrusionJunction(const Point p, const coord_t w, const coord_t perimeter_index, const size_t region_id = 0);
 
     bool operator==(const ExtrusionJunction& other) const;
+
+    void InternalSwap(ExtrusionJunction* other);
```

Comment:
code-style naming we use lowerCamelCase for functions:
```suggestion
    void internalSwap(ExtrusionJunction* other);
```

Reply:
Code was removed.